edx_repo_tools/codemods/django3/github_actions_modernizer_django.py

```diff
@@ -0,0 +1,48 @@
+"""
+Django Matrix Modernizer for Github Actions CI
+"""
+from copy import deepcopy
+
+import click
+
+from edx_repo_tools.utils import YamlLoader
+
+DJANGO_VERSIONS = ['2.2', '3.0', '3.1', '3.2']
+
+
+class GithubCIDjangoModernizer(YamlLoader):
+    def __init__(self, file_path):
+        super().__init__(file_path)
+        self.yml_instance.default_flow_style = None
+        self.yml_instance.indent(mapping=2, sequence=2, offset=0)
+
+    def _update_matrix(self):
+
+        section_key = None
+
+        for key in ['build', 'tests', 'run_tests']:
```

Comment:
these names could be arbitrary, name of the job could be set to anything, we should instead iterate over all the jobs available in the workflow file and check if that job has matrix items set for django